modules/GKB/CommonUtils.pm

```diff
@@ -112,204 +114,259 @@ sub restore_database {
 }
 
 sub database_exists {
     my $db = shift;
     my $host = shift;
-    
+
     my $user = $GKB::Config::GK_DB_USER;
     my $pass = $GKB::Config::GK_DB_PASS;
-    
+
     return (system("mysql -u $user -p$pass -h $host -e 'use $db' 2> /dev/null") == 0);
 }
 
 sub get_name_and_id {
     my $instance = shift;
-    
+
     return '' unless $instance && $instance->is_a('DatabaseObject');
     return $instance->displayName . '(' . $instance->db_id . ')';
 }
 
 sub get_all_species_in_entity {
     my $entity = shift;
     return unless $entity;
-    
+
     my @species;
     push @species, @{$entity->species} if @{$entity->species};
-    
+
     if ($entity->is_a('Complex')) {
         push @species, get_all_species_in_entity($_) foreach @{$entity->hasComponent};
     } elsif ($entity->is_a('EntitySet')) {
         push @species, get_all_species_in_entity($_) foreach (@{$entity->hasMember}, @{$entity->hasComponent});
     }
-    
+
     my %species;
     $species{$_->db_id} = $_ foreach @species;
     return values %species;
 }
 
 sub is_electronically_inferred {
-	my $instance = shift;
-
-	my $dba = $instance->dba();
-	my $db_name = $dba->db_name();
-	return 0 unless $db_name =~ /^test_reactome_\d+$/ || $db_name eq 'gk_current';
-
-	# first, let's check to see if the "magic" note is in the Created InstanceEdit for this object.
-	my $created = $instance->created->[0];
-	if ($created && $created->note && $created->note eq "inferred events based on ensembl compara")
-	{
-		return 1;
-	}
-	# If there's no note to indicate that the object was created during orthoinference,
-	# try the old logic.
-	if ($instance->is_a('Event')) {
-		return $instance->evidenceType->[0] && $instance->evidenceType->[0]->displayName =~ /electronic/i;
-	} elsif ($instance->is_a('PhysicalEntity')) {
-		# If manually inferred physical entities ever start using the inferredFrom slot, this logic
-		# can be augmented to check the event(s) to which the physical entity is/are attached
-		# to see if the event(s) are electronically inferred
-		return $instance->inferredFrom->[0] &&
-			$instance->inferredFrom->[0]->species->[0] &&
-			$instance->inferredFrom->[0]->species->[0]->displayName =~ /^Homo sapiens$/i;
-	} elsif ($instance->is_a('CatalystActivity')) {
-		return any { is_electronically_inferred($_)} @{$instance->reverse_attribute_value('catalystActivity')};
-	} elsif ($instance->is_a('Regulation')) {
-		return any { is_electronically_inferred($_)} @{$instance->reverse_attribute_value('regulatedBy')};
-	}
+    my $instance = shift;
+
+    my $dba = $instance->dba();
+    my $db_name = $dba->db_name();
+    return 0 unless $db_name =~ /^test_reactome_\d+$/ || $db_name eq 'gk_current';
+
+    # first, let's check to see if the "magic" note is in the Created InstanceEdit for this object.
+    my $created = $instance->created->[0];
+    if ($created && $created->note && $created->note eq "inferred events based on ensembl compara") {
```

Comment:
We will want to propagate a change to all the code that will recognize a new note saying that they are based on Panther for release 68

Reply:
Sure, that's a good idea.  Off the top of my head, maybe we could change this code so it looks for the regex pattern of 'inferred events' instead of an exact match to the string that has 'ensembl compara' in it.  

It would be nice to have a more robust way to check that doesn't rely on querying a string value in a note, though.